/**
 * @param {string} val
 * @return {Object}
 */
var expect = function(val) {
    return {
        toBe : (val2) => {
            if (val === val2){
                return true
            }else{
                throw new Error("Not Equal")
            }
        },
        notToBe : (val2) => {
            if (val !== val2){
                return true
            }else{
                throw new Error('Equal')
            }
        }
    }
};

// when there is some error thrown in the code, definitely uses try catch block to catch the errors thrown
try {
    expect(5).toBe(5); 
}catch(error){
    console.log({"error": error})
}

try {
    expect(5).notToBe(5); 
}catch(error){
    console.log({"error": error})
}
